src/sql/workbench/parts/notebook/models/notebookModel.ts

```diff
@@ -920,6 +929,24 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		this._connectionUrisToDispose = [];
 	}
 
+	// Track time it takes to render all markdown cells
+	private trackMarkdownTelemetry(cellContent: nb.ICellContents, cellModel: ICellModel): void {
+		if (cellContent && cellContent.cell_type === CellTypes.Markdown) {
+			this._textCellsLoading++;
+		}
+		this._register(cellModel.onLoaded((e) => {
+			this._textCellsLoading--;
+			if (this._textCellsLoading <= 0) {
+				if (this._notebookOptions.editorLoadedTimestamp) {
+					if (this._notebookOptions.editorLoadedTimestamp && this._notebookOptions.editorLoadedTimestamp > 0) {
```

Comment:
If this check really necessary? If we have a notebookOptions I don't see how this could reasonably be undefined/0. Regardless though you have a duplicate check here for editorLoadedTimestamp

Reply:
Nah, it's not. Thanks for catching.